src/Components/v2/CollectionsHubsHomepageNav.tsx

```diff
@@ -1,41 +1,64 @@
 import { CSSGrid } from "@artsy/palette"
 import { Serif } from "@artsy/palette"
 import { CollectionsHubsHomepageNav_marketingHubCollections } from "__generated__/CollectionsHubsHomepageNav_marketingHubCollections.graphql"
-import React, { FC } from "react"
+import { AnalyticsSchema } from "Artsy/Analytics"
+import React, { useEffect } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import track, { useTracking } from "react-tracking"
 import { resize } from "Utils/resizer"
 import { ImageLink } from "./ImageLink"
 
 interface CollectionsHubsHomepageNavProps {
   marketingHubCollections: CollectionsHubsHomepageNav_marketingHubCollections
 }
 
-export const CollectionsHubsHomepageNav: FC<
-  CollectionsHubsHomepageNavProps
-> = props => {
-  return (
-    <CSSGrid
-      as="aside"
-      gridTemplateColumns={[
-        "repeat(2, 1fr)",
-        "repeat(2, 1fr)",
-        "repeat(3, 1fr)",
-      ]}
-      gridGap={20}
-    >
-      {props.marketingHubCollections.slice(0, 6).map(hub => (
-        <ImageLink
-          to={`/collection/${hub.slug}`}
-          src={resize(hub.thumbnail, { width: 357, height: 175 })}
-          ratio={[0.49]}
-          title={<Serif size={["3", "4"]}>{hub.title}</Serif>}
-          subtitle={<Serif size="2">{subtitleFor(hub.title)}</Serif>}
-          key={hub.id}
-        />
-      ))}
-    </CSSGrid>
-  )
-}
+export const CollectionsHubsHomepageNav = track()(
+  (props: CollectionsHubsHomepageNavProps) => {
+    const { trackEvent } = useTracking()
+    useEffect(() => {
+      trackEvent({
+        action_type: AnalyticsSchema.ActionType.Impression,
+        context_page: AnalyticsSchema.PageName.HomePage,
+        context_module: AnalyticsSchema.ContextModule.CollectionHubEntryPoint,
+        subject: AnalyticsSchema.Subject.FeaturedCategories,
+      })
+    }, [])
```

Comment:
Is this mystery `[]` argument the thing that keeps this impression event from firing on subsequent renders _after_ mounting?